core/localization.lua

```diff
@@ -10,15 +10,133 @@ local L = addon:NewModule('Localization')
 -- Data is set outside of the initialization function so that
 -- it loads when the file is read.
 L.data = {}
+local locale = GetLocale()
+if locale == 'frFR' then
+L.data["Backpack"] = "Sac à dos"
+L.data["Left Click"] = "Clic Gauche"
+L.data["Open Menu"] = "Ouvre le Menu"
+L.data["Close Menu"] = "Ferme le Menu"
+L.data["Bank"] = "banque"
+L.data["Shift Left Click"] = "Shift clic gauche"
+L.data["Show Bags"] = "Afficher les sacs"
+L.data["Show Currencies"] = "Afficher les devises"
+L.data["Welcome to Better Bags! Please select a help item from the left menu for FAQ's and other information."] = "Bienvenue chez Better Bags ! Veuillez sélectionner un élément d'aide dans le menu de gauche pour accéder aux FAQ et à d'autres informations."
+
+L.data["Click to toggle the display of the bag slots."] = "Cliquez pour basculer l’affichage des emplacements de sacs."
+L.data["Click to toggle the display of the currencies side panel."] = "Cliquez pour basculer l’affichage du panneau latéral des devises."
+
+L.data["Open Options Screen"] = "Ouvrir l'écran des options"
+L.data["Click to open the options screen."] = "Cliquez pour ouvrir l'écran des options"
+
+
+-- Help
+L.data["Custom Categories"] = "Catégories personnalisées"
+L.data["Categories"]="Catégories"
+L.data["Select which categories to show in this bag. If an option is checked, items that belong to the checked category will be put into a section for that category."]="Sélectionnez les catégories à afficher dans ce sac. Si une option est cochée, les éléments appartenant à la catégorie cochée seront placés dans une section pour cette catégorie."
+
```

Comment:
L.data[ [=[
    You can bind a key to open the search bar in the World of Warcraft keybindings menu. You can also open the search bar by shift clicking on the bag button at the top of the bag frame.
    Once the search bar is open, you can type in the name of an item to search for it. You can also use the following search operators:
    - `type:` to search for items of a specific type (e.g. `type:weapon`)
    - `subtype:` to search for items of a specific subtype (e.g. `subtype:axe`)
    - `name:` to search for items with a specific name (e.g. `name:axe`)
    - `gear:` to search for items that are part of an equipment set (e.g. `gear:tank`)
    - `exp:` to search for items from a specific expansion (e.g. `exp:shadowlands`)
    
    For expansions, you can use the following expansions shortcuts (e.g. `exp:classic`):
    - `classic`
    - `tbc`
    - `wotlk`
    - `cata`
    - `mop`
    - `wod`
    - `legion`
    - `bfa`
    - `shadowlands`
    - `dragonflight`
  ]=] ] = 
  [=[Vous pouvez lier le mot clé pour ouvrir la barre de recherche dans le menu des raccourcis clavier de World of Warcraft. Vous pouvez également ouvrir la barre de recherche en cliquant sur le bouton du sac en haut du cadre du sac.
  Une fois la barre de recherche ouverte, vous pouvez saisir le nom d'un élément pour le rechercher. Vous pouvez également utiliser les opérateurs de recherche suivants :
  - `type:` pour rechercher des objets d'un type spécifique (par exemple `type:weapon`)
  - `subtype:` pour rechercher des éléments d'un sous-type spécifique (par exemple `subtype:axe`)
  - `name:` ​​pour rechercher des éléments avec un nom spécifique (par exemple `name:axe`)
  - `gear:` pour rechercher des éléments faisant partie d'un ensemble d'équipement (par exemple `gear:tank`)
  - `exp:` pour rechercher des éléments d'une extension spécifique (par exemple `exp:shadowlands`)
  
  Pour les extensions, vous pouvez utiliser les raccourcis d'extension suivants (par exemple `exp:classic`) :
  - 'classique'
  - `à confirmer`
  - 'Wotlk'
  - `cata`
  - `vadrouille`
  - 'Wod'
  - 'légion'
  - 'bfa'
  - 'pays de l'ombre'
  - `vol du dragon`
  ]=]